src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponentSpec.scala

```diff
@@ -317,21 +328,28 @@ class ClusterComponentSpec extends TestComponent with FlatSpecLike with CommonTe
       Some(GcsBucketName("testStagingBucket1")),
       List.empty,
       Set(masterInstance, workerInstance1, workerInstance2),
-      None
+      None,
+      dateaccessed
     )
 
     dbFutureValue { _.clusterQuery.save(c1, gcsPath( "gs://bucket1"), Some(serviceAccountKey.id)) } shouldEqual c1
     // note: this does not update the instance records
     dbFutureValue { _.clusterQuery.setToStopping(c1.googleId) } shouldEqual 1
-    dbFutureValue { _.clusterQuery.getByGoogleId(c1.googleId) } shouldEqual Some(
+    val c2: Option[Cluster] = dbFutureValue { _.clusterQuery.getByGoogleId(c1.googleId) }
+    c2.map(_.copy( dateAccessed = dateaccessed)) shouldEqual Some(
```

Comment:
Nit: remove the space between `copy(` and  `dateAccessed` and camelCase `dateaccessed`. In fact, why not just call `Instant.now()` directly? Same below.

Reply:
Instant.now() would return different values in both cases, that would break the tests.